massa-execution/src/vm.rs

```diff
@@ -409,6 +451,12 @@ impl VM {
             let block_creator_addr = Address::from_public_key(&block.header.content.creator);
             // run all operations
             for (op_idx, operation) in block.operations.iter().enumerate() {
+                // get operation ID
+                // TODO for the refactoring, have operations include their operationID (without necessarily serializing it)
```

Comment:
Please open an issue or add it to an existing issue.

Reply:
see #2234